english/run_out_the_door/run_out_the_door.md

```diff
@@ -2,6 +2,9 @@ You immediately jump up and run out the nearest exit!
 
 You immediately jump up and run out the nearest [exit](../new-exit/new-exit.md)!
 
+You find that a mysterious pool has formed right outside the door and with a splash
+you [fall in](../mysterious-pool/mysterious-pool.md)!
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.